packages/webdriver/src/utils.ts

```diff
@@ -184,7 +190,7 @@ export function getPrototype ({ isW3C, isChrome, isMobile, isSauce, isSeleniumSt
  * @param  {Object} body body object
  * @return {Object} error
  */
-export function getErrorFromResponseBody (body) {
+export function getErrorFromResponseBody (body: any) {
```

Comment:
is it really any?

Reply:
yeah, we never gonna know what the driver returns

Reply:
maybe it's unknown then?

Reply:
mhm .. setting it unknown makes TS to cast it to be an `object` that doesn't have `value` or `error` as properties, I would keep it as `any` as there is no type advantage setting it to unknown and then manually casting it again.

Reply:
marking as resolved